### PR TITLE
Android Translucent Support

### PR DESCRIPTION
**Pull request by iamacup**
Hi - i had a pull request before but never got around to properly explaining the change:

This is slightly modified from last time as it now accepts the statusBarIsTranslucent prop for showMessage.

When you are running on an ejected react native project - the current accommodation for a translucent status bar does not work because it relies on the Expo global. There is currently no way to detect if an ejected project has a translucent status bar on android - and if there is a translucent status bar - the content of a flash message with position = "top" will underflow the status bar slightly.

In order to combat this in all cases / status bar heights / notches, we add an override - `statusBarIsTranslucent` - that when set to true will set the top padding to the height of the status bar on android.

Here is an example with images:

```
this.flash.showMessage({
    message: 'Some title',         
    description: 'Some content',
    duration: 10000,
    position: 'top',
    type: 'default',
    backgroundColor: 'red',
    color: 'white',
    statusBarIsTranslucent: false,  // can be not set as well, defaults to false
});
```

https://imgur.com/OoaDdOY

```
this.flash.showMessage({
    message: 'Some title',         
    description: 'Some content',
    duration: 10000,
    position: 'top',
    type: 'default',
    backgroundColor: 'red',
    color: 'white',
    statusBarIsTranslucent: true,  
});
```

https://imgur.com/2roVtDv

So in short, when using `<StatusBar translucent />` or `StatusBar.setTranslucent(true);` the `statusBarIsTranslucent` property for `showMessage` will prevent content in flash messages being rendered under the status bar.

### CODE REVIEW

**Comment by iamacup** [comment]
out of date now
